test_framework/test_jit.py

```diff
@@ -49,6 +49,10 @@ def check_datafile(filename):
             cmd = [VM]
             if memfile:
                 cmd.extend(['-m', memfile.name])
+            if 'reload' in data:
```

Comment:
Do we need a test_common.py and refactor this?

Reply:
Hmm maybe, but that would fall outside of this PR I think